src/Bundle/Parser/IniParser.php

```diff
@@ -36,7 +36,7 @@ public function __construct(string $modulesDir)
      */
     public function parse($resource, $type = null): array
     {
-        if (isset($this->loaded[$resource])) {
+        if (isset($this->loaded[$resource]) || !is_dir($this->modulesDir.'/'.$resource)) {
```

Comment:
```suggestion
        if (isset($this->loaded[$resource]) || !is_dir(Path::join($this->modulesDir, $resource))) {
```

This should be safer (use statement for Filesystem\Path still missing though).

Reply:
I thought about this too, but all the other lines in this file do not use that so I thought its not worth it 🙈

Reply:
Unfortunately I cannot resolve the conversation as I am not added as a maintainer here.